### PR TITLE
Network and refactoring

### PR DESCRIPTION
**Pull request by abhianuj**
This PR contains:
1. Order creation
2. Dio implementation
3. CRUD calls config without TLS/SSL

### CODE REVIEW

File: App/app/lib/api/user_requests.dart [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  dio.options.headers["x-approuter-authorization"] = "Bearer $token";
+
+  Response<dynamic> response = await dio.get("/user");
+
+  print(response.data.toString());
+}
+
+Future<void> sendOrder(List<Map<String, dynamic>> orderItems) async {
+  // remove afterwards
```
Comment (abhianuj) [inline App/app/lib/api/user_requests.dart:30]: This will be removed in a new PR, when login is implemented.

**Review by gourabatsap — APPROVED** [review]
LGTM because I won't L 🤣

File: App/app/android/build.gradle [hunk 2]
```diff
@@ -26,6 +26,6 @@ subprojects {
     project.evaluationDependsOn(':app')
 }
 
-task clean(type: Delete) {
+tasks.register("clean", Delete) {
     delete rootProject.buildDir
```
Comment (gourabatsap) [inline App/app/android/build.gradle:30]: Interesting how you made me delete in my PR and pushed it yourself 🤨

**Comment by kumarniti4** [comment]
GJ merging

**Pull request merged**